_pkgdown.yml

```diff
@@ -49,13 +49,14 @@ reference:
     desc: Utility functions that are used in other cchsflow functions
     contents:
     - if_else2
-  - title: "Alcohol CCHS variables"
-    desc: Derived alcohol variables in cchsflow that users may run into issues with
+  - title: "Potentially problematic variables"
```

Comment:
Instead of creating a separate section for food_insecurity_der, I've added to the list of alcohol variables that have been documented and renamed the section "Potentially problematic variables". @DougManuel @DarkShadeKnigh thoughts?

Reply:
Makes sense